lib/ts/ink_code.h

```diff
@@ -36,9 +37,8 @@ typedef MD5_CTX INK_DIGEST_CTX;
 */
 
 inkcoreapi int ink_code_md5(unsigned const char *input, int input_length, unsigned char *sixteen_byte_hash_pointer);
-inkcoreapi char *ink_code_to_hex_str(char *dest33, uint8_t const *md5);
```

Comment:
If you are going to remove this, you should be able to remove the actual function in `ink_code.cc`.

Reply:
Done